tests/keras/test_keras_model_export.py

```diff
@@ -210,21 +210,30 @@ def _import_module(name, **kwargs):
             assert x == b
 
 
-@pytest.mark.skip("Unskip this test once https://github.com/h5py/h5py/issues/1732 is fixed")
-@pytest.mark.parametrize("build_model", [model, tf_keras_model])
+@pytest.mark.parametrize(
+    "build_model,save_format",
+    # Unskip these parameters once https://github.com/h5py/h5py/issues/1732 is fixed
+    # [(model, None), (tf_keras_model, None), (tf_keras_model, "h5"),
+    [(tf_keras_model, "tf")],
+)
 @pytest.mark.large
-def test_model_save_load(build_model, model_path, data):
+def test_model_save_load(build_model, save_format, model_path, data):
     x, _ = data
     keras_model = build_model(data)
     if build_model == tf_keras_model:
         model_path = os.path.join(model_path, "tf")
     else:
         model_path = os.path.join(model_path, "plain")
     expected = keras_model.predict(x)
-    mlflow.keras.save_model(keras_model, model_path)
+    kwargs = {"save_format": save_format} if save_format else {}
+    mlflow.keras.save_model(keras_model, model_path, **kwargs)
     # Loading Keras model
     model_loaded = mlflow.keras.load_model(model_path)
-    assert type(keras_model) == type(model_loaded)
+    # When saving as SavedModel, we actually convert the model
+    # to a slightly different format, so we cannot assume it is
+    # exactly the same.
+    if save_format is not "tf":
```

Comment:
```suggestion
    if save_format != "tf":
```

should we use != here?